site/content/docs/developing/domain.en.md

```diff
@@ -91,7 +103,7 @@ http:
   alias: example.aws
   hosted_zone: Z0873220N255IR3MTNR4
 ```
-2. Deploy the service without the `hosted_zone` field, then manually add the DNS name of the Application Load Balancer (ALB) created in that environment as an A record where your alias domain is hosted.
+2. Alternatively, deploy the service without the `hosted_zone` field, then manually add the DNS name of the Application Load Balancer (ALB) created in that environment as an A record where your alias domain is hosted.
```

Comment:
Is this redundant with the "you can either" above? Maybe drop "either" and take out the numbering? Not sure what makes the most sense here.

Reply:
> Maybe drop "either" and take out the numbering? 

Done!